diesel/src/sql_types/mod.rs

```diff
@@ -285,13 +285,21 @@ pub struct Date;
 /// ### [`ToSql`](../serialize/trait.ToSql.html) impls
 ///
 /// - [`PgInterval`] which can be constructed using [`IntervalDsl`]
+/// - [`std::time::Duration`](https://doc.rust-lang.org/std/time/struct.Duration.html)
+///   [^microseconds]
+/// - [`chrono::Duration`](/chrono/struct.Duration.html)
```

Comment:
This link will break on docs.rs.

Reply:
This means that the docs for `Time` and `Timestamp` are also broken, because they do the same. 
What is the preferred way to handle this? Just link explicitly to docs.rs?

Reply:
Can it not just be `[chrono::Duration]` with backticks? ([eg](https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md#implied-shortcut-reference-links))